src/components/bulkConcepts/component/ConceptPagination.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Paginations from './Pagination';
+import { conceptsProps } from '../../dictionaryConcepts/proptypes';
+
+export class ConceptPagination extends Component {
+  static propTypes = {
+    concepts: PropTypes.arrayOf(PropTypes.shape(conceptsProps)).isRequired,
+    currentPage: PropTypes.number.isRequired,
+    handleNextPage: PropTypes.func, // eslint-disable-line react/require-default-props
```

Comment:
Why I disable the error inside of add default props was because handleNextPage is a function. To include a default prop will be to have something like this 
```
static defaultProps = {
    handleNextPage: () => null,
  }
```
And it will need to be tested since it is a  function,

Reply:
Ok.

Reply:
There won't be need for default if you can make it required `handleNextPage: PropTypes.func.isRequired`. Except there is a reason you can't

Reply:
It throws a console error on BulkConceptsPage.test.jsx which is parent to ConceptPagination.test.jsx. Which @abulojoshua1  already discussed with us this morning to make sure our tests are not showing those warning error.